python/tank/path_cache.py

```diff
@@ -644,9 +662,17 @@ def _replay_folder_entities(self, cursor, log, max_event_log_id, ids=None):
                                    SG_ENTITY_TYPE_FIELD, 
                                    SG_ENTITY_NAME_FIELD],
                                   [{"field_name": "id", "direction": "asc"},])
+        elif ids == []:
```

Comment:
interesting style choice ;) - would have probably written it like this:

elif len(ids) == 0: